packages/components/organisms/f-offers/src/components/NoResults.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <div
+        :class="$style['c-noResults-wrapper']"
+    >
+        <media-element
+            :title="$t('noResults.title')"
+            :text="$t('noResults.subtitle')"
+            image-url="https://d30v2pzvrfyzpo.cloudfront.net/a/hw/img/decoration/oi_no-results-image.svg"
```

Comment:
I'm assuming this is in lieu of an agreed cloudinary upload location?

Reply:
Yea I am still waiting on a reply to my message in help-content, will chase up again today